queries.sql

```diff
@@ -110,4 +110,27 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT MAX(escape_attempts) FROM animals WHERE neutered = false OR true;
 
 --What is the minimum and maximum weight of each type of animal?
-SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
\ No newline at end of file
+SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
+
+--What animals belong to Melody Pond?
+SELECT name FROM animals INNER JOIN owners ON animals.owners_id = owners.owners_id WHERE owners.full_name = 'Melody';
+
+--List of all animals that are pokemon (their type is Pokemon).
+SELECT * FROM animals INNER JOIN species ON animals.species_id = species.spcies_id WHERE species.species_name = 'Pokemon';
+
+--List all owners and their animals, remember to include those that don't own any animal.
+SELECT * FROM animals RIGHT JOIN owners ON animals.owners_id = owners.owners_id;
+
+--How many animals are there per species?
+SELECT COUNT(name) FROM animals INNER JOIN species ON animals.species_id = species.spcies_id WHERE species.species_name = 'Pokemon';
+
+SELECT COUNT(name) FROM animals INNER JOIN species ON animals.species_id = species.spcies_id WHERE species.species_name = 'Digimon';
```

Comment:
- [ ] Great job so far @imranhossainemi  Overall, you've made good progress, and I appreciate your efforts :wink:. Let's discuss a specific area that needs attention. :100: 
- **Your Current Query Results** 
![image](https://github.com/imranhossainemi/vet_clinic_database/assets/102699958/3f648f5d-1fb4-440a-a196-05b456cbdb57)

- [ ] I noticed that In your query, you attempted to count the number of animals per species. However, the `join` condition and `filtering` are not correctly implemented. Instead of using the appropriate `join` and `grouping` syntax, you used a `WHERE` clause to filter by species name. This approach can lead to incorrect results and limit the flexibility of the query. 
- [ ] Currently, you only have the number of `pokemon` in your screenshot results  but you should have the number of both species in the screenshot :+1: 
- [ ] To achieve accurate results and follow best practices, I recommend using the following approach:

```
SELECT species.name as species_name, COUNT(*) as num_animals
FROM animals
JOIN species ON animals.species_id = species.id
GROUP BY species.name;

```
- This query joins the `animals` and `species` tables based on the `species ID`, and then groups the results by `species name`, allowing you to `count` the number of animals per species accurately. :wink: 

- I encourage you to make these adjustments in your code to ensure the query provides the desired outcome and follows standard practices. If you have any questions or need further clarification, please don't hesitate to ask. I'm here to help!
- [ ] KIndly include the screenshot of this query in your pull request afterwards :100: